docs/reference/commandline/dockerd.md

```diff
@@ -780,6 +785,113 @@ conditions the user can pass any size less then the backing fs size.
 $ sudo dockerd -s overlay2 --storage-opt overlay2.size=1G
 ```
 
+
+#### Windowsfilter options
+
+##### `size`
+
+Specifies the size to use when creating the sandbox which is used for containers.
+Defaults to 20G.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt size=40G
+```
+
+#### LCOW (Linux Containers on Windows) options
+
+##### `lcow.globalmode`
```

Comment:
Ultimately it's the storage driver that instantiates and manages the lifetime of all the utility VMs for the remote file system operations. Storage opts are the only thing that get this down to the driver (and also are present for `run`). I'm really not sure you'd want me changing graphdriver interfaces....

However, this is documentation, and documenting what's implemented and in master.... If you strongly think a code change is necessary (I'm far from convinced), then that should be a separate issue on moby/moby.